src/network/networkd-link.c

```diff
@@ -2588,14 +2588,6 @@ static int link_configure(Link *link) {
                 return 0;
         }
 
-        /* Drop foreign config, but ignore loopback or critical devices.
```

Comment:
I wonder whether this commit is still necessary now that networkd supports unmanaged interfaces.

Reply:
I'll drop it.  We should state this behavior change in the release notes.